airbyte-commons-protocol/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     testImplementation libs.bundles.micronaut.test
 
     implementation project(':airbyte-protocol:protocol-models')
+    implementation project(':airbyte-json-validation')
```

Comment:
added this dependency because the downgrade migration needs to be able to test whether a record (or part of a record) matches a schema (or subschema).